### PR TITLE
Facelift of README

### PR DESCRIPTION
**Pull request by honzajavorek**
I needed to install `apiary-client` locally and I realized the README is so terrible and unhelpful that I couldn't prevent myself to initiate a change.

**Note:** https://github.com/apiaryio/apiary-client/issues/78 should be resolved before merging this.

### CODE REVIEW

**Comment by honzajavorek** [comment]
**Issue:** output of `apiary help` is following:

```
$ apiary help
Commands:
  apiary fetch --api-name=API_NAME    # Fetch apiary.apib from API_NAME.apiary.io
  apiary help [COMMAND]               # Describe available commands or one specific command
  apiary preview                      # Show API documentation in default browser
  apiary publish --api-name=API_NAME  # Publish apiary.apib on docs.API_NAME.apiary.io
  apiary version                      # Show version
```

That's very far from the version in the original README:

```
$ apiary help

Usage: apiary command [options]
Try 'apiary help' for more information.

Currently available apiary commands are:

  preview                                     Show API documentation in default browser
  preview --browser [chrome|safari|firefox]   Show API documentation in specified browser
  preview --output [FILE]                     Write generated HTML into specified file
  preview --path [PATH]                       Specify path to blueprint file
  preview --api_host [HOST]                   Specify apiary host
  preview --server                            Start standalone web server on port 8080
  preview --server --port [PORT]              Start standalone web server on specified port
  publish --api-name [API_NAME]               Publish apiary.apib on docs.API_NAME.apiary.io
  publish --api-name [API_NAME] \
          --message [COMMIT_MESSAGE]          Publish with custom commit message
  fetch   --api-name [API_NAME]               Fetch apiary.apib from API_NAME.apiary.io
          --output [FILE]                     Write apiary.apib into specified file
  help                                        Show this help
  version                                     Show version
```

I see there's even already a dedicated issue for that from @Almad: https://github.com/apiaryio/apiary-client/issues/78

File: README.md [hunk 1]
```diff
@@ -1,81 +1,91 @@
...
+Inside the `apiary-client` repository directory run:
+
+```sh
+$ bundle exec rake test
+$ bundle exec rake features
+```
+
+### Release
+
+Use `bundle install` to test locally released version. Use `gem push` to
```
Comment (abtris) [inline README.md]: here you can use `rake install` to build and test gem before `rake release` to make tag and push into rubygems

**Comment by abtris** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (honzajavorek) [inline README.md]: I didn't see these tasks in `Rakefile`. What kind of magic is this? :dizzy:

Comment (abtris) [inline README.md]:
```
$ rake -T
rake build     # Build apiaryio-0.3.0.gem into the pkg directory
rake default   # Default: Run all specs
rake features  # Run Cucumber features
rake gemspec   # Validate the gemspec
rake install   # Build and install apiaryio-0.3.0.gem into system gems
rake release   # Create tag v0.3.0 and build and push apiaryio-0.3.0.gem to Rubygems
rake spec      # Run all specs
rake yard      # Generate YARD Documentation
```
Not magic just include

**Comment by jrep** [comment]
I have made #96, which I believe encompasses and obsoletes this PR.

**Pull request closed without merging**